templates/ui-service.yaml

```diff
@@ -1,7 +1,10 @@
 {{ template "vault.mode" . }}
+# {{ .mode }}
 {{- if ne .mode "external" }}
-{{- if and (ne .mode "") (eq (.Values.global.enabled | toString) "true") }}
-{{- if eq (.Values.ui.enabled | toString) "true" }}
+{{- template "vault.uiEnabled" . -}}
+# {{ .uiEnabled }}
```

Comment:
Still needed?

```suggestion
```

Reply:
Thanks for catching these. I was using them in debugging :)